tier4_planning_msgs/msg/PlanningFactor.msg

```diff
@@ -0,0 +1,24 @@
+# msg format version
+string version_msg_format
+
+# constants for behavior
+uint16 UNKNOWN = 0
+uint16 SLOW_DOWN = 1
+uint16 STOP = 2
+uint16 GO = 3
+uint16 LEFT_SHIFT = 4
+uint16 RIGHT_SHIFT = 5
+uint16 STRAIGHT = 6
```

Comment:
What is the difference between GO and STRAIGHT?

Reply:
GO is the longitudinal behavior. e.g. it's used when the ego can pass the crosswalk safely without stopping.
STRAIGHT is the lateral behavior. e.g. it's used when the ego keeps following current lane.

But I feel that there is better name... :thinking:

Reply:
I renamed `NONE` according to your advice.